.github/workflows/build-test.yaml

```diff
@@ -15,12 +16,29 @@ jobs:
             gofmt -s -d .
             exit 1
           fi
-      - name: Build Tracee
-        run: make all DOCKER=1
-  test:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout Code
-        uses: actions/checkout@v2
+
+      - name: yaml-lint
+        uses: ibiqlik/action-yamllint@v3
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
```

Comment:
correct me if I'm wrong but won't this cache only between builds on the same commit?
we want to be able to cache across commits

Reply:
Using this settings, we are doing cache across commits,
```
          cache-from: type=local,src=/tmp/.buildx-cache
          cache-to: type=local,dest=/tmp/.buildx-cache,mode=max
```